api/workAllocation2/interfaces/caseRole.ts

```diff
@@ -0,0 +1,14 @@
+export interface CaseRole {
+  name: string;
+  role: string;
+  location: string;
+  start: string;
+  end: string;
```

Comment:
Minor query is we already have a Location interface - is this not related to this? Also, would start and end not be Date types rather than string? Just wanted to ask in case this has been done quick and not considered

Reply:
@connorpgpmcelroy I am not sure at his moment of time. I just assumed that the location property will be a `string`. No  one has mentioned the type for `location`.

Reply:
As it's the Node layer not sure it will be relevant anyway - can always be changed if needed later